src/scopes/activityReport/index.test.js

```diff
@@ -118,13 +116,6 @@ describe('filtersToScopes', () => {
     }, {
       silent: true,
     });
-    // Create ES client.
```

Comment:
I moved this ES stuff into the test that requires it, as it was kind of noisy where it was when I was working on this test